packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -41,24 +40,15 @@ const AppContents = observer(({ children }) => {
 
     const location = useLocation();
 
-    const current_page = window.location.hostname + window.location.pathname;
-
     React.useEffect(() => {
-        if (is_logged_in && user_id) {
-            const { tracking } = Analytics?.getInstances();
-            Analytics?.setAttributes({
-                account_type: loginid.substring(0, 2),
-            });
-            tracking?.identifyEvent(user_id, {
-                language: getLanguage().toLowerCase() || 'en',
-            });
-            Analytics?.pageView(current_page);
-        }
         if (scroll_ref.current) setAppContentsScrollRef(scroll_ref);
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    Analytics?.pageView(current_page);
+    React.useEffect(() => {
+        Analytics.pageView(window.location.href);
+        // react-hooks/exhaustive-deps
+    }, [window.location.href]);
```

Comment:
I think the better place to put this is inside the route component

Reply:
No @amir-deriv I checked your prev implementation. It was wrong. To minify unecessarry calls we should calls pageView only like that. Or in more proper place